### PR TITLE
Modified README.md and found a bug

### PR DESCRIPTION
**Pull request by Axwaizee**
modified the readme file where in Installation part

it was mentioned

```bash
git clone git@github.com:Thenlie/readme-generator.git
```

where many users don't use SSH so they may get error and end up with nothing. So, I added an OR operator 

```bash
git clone git@github.com:Thenlie/readme-generator.git || git clone https://github.com/Thenlie/readme-generator.git
```

now if the first command gets error then the 2nd one will surely work.
Also, while editing and replicating the instruction for installation and running the application i found that the 
command 

```bash
node index.js
```

won't work because the index.js file is no more on the root directory

image

it's in the `/dist` directory.

Also, while executing or creating my `README.md` file i used `npm start` **NOT** `node index.js`.

image

You have already described the Usage in Installation part with `npm start` and in the Usage portion is wrong so you make changes to it accordingly. Maybe its because you never updated the readme file later also the gif/video in the Usage part needs to be updated.

Thank you
Ahh! typed so long hope they gets fixed soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,11 +45,21 @@
...
+  ```bash
+  npm install
   ```
   
-  You can then use the command `npm install` to download the projects npm dependencies. Currently, that is Inquirer, TypeScript and Es-Lint. Once you have done that you are ready to compile the typescript. This can be done with the command `npm run build`. This will output JavaScript files to the `/dist` directory which are used by node. Finally, to run the application, use the command `npm start`. 
+  to download the projects npm dependencies. Currently, that is Inquirer, TypeScript and Es-Lint. Once you have done that you are ready to compile the typescript. This can be done with the command `npm run build`. This will output JavaScript files to the `/dist` directory which are used by node. Finally, to run the application, use the command
+  
+  ```bash
+  npm start
+  ```
```
Comment (Thenlie) [inline README.md:62]: I think this step is fine, it is the Usage step that has the error.

File: README.md [hunk 2]
```diff
@@ -45,11 +45,21 @@
 
   To run this program simply clone the repo from GitHub using the following command, 
   
+  ```bash
+  git clone git@github.com:Thenlie/readme-generator.git || git clone https://github.com/Thenlie/readme-generator.git
```
Comment (Thenlie) [inline README.md:49]: I think I would prefer something like:
---
Use the steps below to clone the repository and get the project running on your local machine.

1. Navigate to the develop branch of the repository and click the green "Code" button. Then copy the repository URL with the method of your choosing. We suggest SSH which can be setup be following [these docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).

2. Go to your terminal program of choice and navigate the the folder you want the project to live in. Then run the command below to clone the repository:

```sh
git clone <repository_url>
```
---
This let's the user know that we are showing steps for SSH, but still gives them the flexibility to choose HTTPS.

**Review by Thenlie — CHANGES_REQUESTED** [review]
Thank you for the feedback here! Definitely missed some updates in the README after converting this project to TypeScript. 

I think the main error you pointed out is actually missing from this PR. The "Usage" step should be updated to say, "After using the `npm start` command..."

**Comment by Axwaizee** [comment]
Okay, I made various changes to the file, you may review and merge it. If anything need to be corrected or modified then let me know.

Thank you.

**Review by Thenlie — CHANGES_REQUESTED** [review]
These changes break the table of contents since they alter the markdown headings. I think I would prefer to stick to the initial commit and just update the one or two things pointed out in the first review. I don't think a comprehensive guide on how to copy the project is necessary since GitHub already has great documentation on how to do that. If you disagree, I am happy to hear your reasoning.

**Comment by Axwaizee** [comment]
Have a look at this:

image

If this is alright then you may go with PR.

Any other changes would be appreciated.

**Review by Thenlie — APPROVED** [review]

**Comment by Thenlie** [comment]
@Axwaizee Sorry I forgot about this! 😅

**Pull request merged**